src/alfalfa/Alfalfa.i

```diff
@@ -34,7 +34,7 @@
   using namespace openstudio::alfalfa;
 %}
 
-%ignore openstudio::alfalfa::AlfalfaComponentBase;
+%ignore openstudio::alfalfa::AlfalfaComponentBase::clone;
```

Comment:
@TShapinsky not sure why it was entirely ignored, or if it was just a typo?

Not ignoring it doesn't mean you can instantiate it.

```python
In [2]: openstudio.alfalfa.AlfalfaComponentBase()
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
Cell In[2], line 1
----> 1 openstudio.alfalfa.AlfalfaComponentBase()

File /media/DataExt4/Software/Others/OS-build-release2/Products/python/openstudioalfalfa.py:141, in AlfalfaComponentBase.__init__(self, *args, **kwargs)
    140 def __init__(self, *args, **kwargs):
--> 141     raise AttributeError("No constructor defined - class is abstract")

AttributeError: No constructor defined - class is abstract
```